anta/catalog.py

```diff
@@ -158,34 +155,38 @@ def flatten_modules(data: dict[str, Any], package: str | None = None) -> dict[Mo
                 test_definitions: list[AntaTestDefinition] = []
                 for test_definition in tests:
                     if not isinstance(test_definition, dict):
-                        raise ValueError(f"Syntax error when parsing: {test_definition}\nIt must be a dictionary. Check the test catalog.")
+                        msg = f"Syntax error when parsing: {test_definition}\nIt must be a dictionary. Check the test catalog."
+                        raise ValueError(msg)
                     if len(test_definition) != 1:
+                        msg = (f"Syntax error when parsing: {test_definition}\n"
+                               "It must be a dictionary with a single entry. Check the indentation in the test catalog.")
                         raise ValueError(
-                            f"Syntax error when parsing: {test_definition}\nIt must be a dictionary with a single entry. Check the indentation in the test catalog."
+                            msg,
```

Comment:
this looks overkill is there a good reason?
Plus why the comma

Reply:
For the comma: https://docs.astral.sh/ruff/rules/missing-trailing-comma/
For the msg var, need to check which Ruff rule it is

Reply:
well the coma is usless here just `raise ValueError(msg)` no ?

Reply:
Reason for the msg var: https://docs.astral.sh/ruff/rules/f-string-in-exception/